### ISSUE TITLE
Any sugestions for training with small image size 256*320

### ISSUE THREAD

**Issue by DoubleWJX** [NVlabs/FoundationStereo#61]
Hello, I have tried to fine-tune FoundationStereo model from your weights on my own datasets. The sequence loss is from https://github.com/Junda24/MonSter/blob/main/train_eth3d.py#L57. Due to the gpu memory is 40G and to save training time, I changed the image size from 320 * 736 as you mentioned in paper to 256 * 320. However, when I train about 2 - 3 epochs, Nan and Inf will happen in pred_disps. But if I change the image size to a larger one like 388 * 480, no NAN and INF anymore. 

I want to ask that when I change the image shape to 256*320, should I change some other codes as well? And do you have any ideas to avoid nan or inf in pred_disps? Thank you!

**Comment by Eirikalb** [comment]
I have observed the same, have tried getting up training for the booster dataset but my result just do not seem to work. After inspecting the internal prediction just breaks down: There are transformer parts of these models that could be more sensitive to dimensions compared to CNN architecture, but it could also be as simple as the small image size meaning that there are just too few matching features as the disparity is close to or bigger than than the image width. Disparity is capped to 420 which is above the width. Have you tried playing with the batch size instead of image size? You should be able to at least do one image at 320*736 with 40G right?

![Image](https://github.com/user-attachments/assets/21ff30d8-cbe9-40e4-88ce-736065d7c67f)

**Comment by DoubleWJX** [comment]
> I have observed the same, have tried getting up training for the booster dataset but my result just do not seem to work. After inspecting the internal prediction just breaks down: There are transformer parts of these models that could be more sensitive to dimensions compared to CNN architecture, but it could also be as simple as the small image size meaning that there are just too few matching features as the disparity is close to or bigger than than the image width. Disparity is capped to 420 which is above the width. Have you tried playing with the batch size instead of image size? You should be able to at least do one image at 320*736 with 40G right?
> 
> ![Image](https://github.com/user-attachments/assets/21ff30d8-cbe9-40e4-88ce-736065d7c67f)

No, I haven't try that. but your are right that 40G is enough for one single image ad 320 * 736
